apps/web/src/app/mevchecker/analyzing/layout.tsx

```diff
@@ -0,0 +1,57 @@
+"use client";
+
+import Footer from "@/components/Footer";
+import MainWrapper from "@/components/MainWrapper";
+import Header from "@/components/Header";
+import { useRef, useEffect } from "react";
+import { createPortal } from "react-dom";
+
+function BackgroundRevealElement() {
+  const element = useRef<HTMLDivElement | null>(null);
+  useEffect(() => {
+    function handler(e: MouseEvent) {
+      if (element.current) {
+        const x = e.clientX,
+          y = e.clientY;
+        element.current.style.visibility = "visible";
+        element.current.style.clipPath = `circle(200px at ${x}px ${y}px)`;
+        element.current.style.maskImage = "url(/images/reveal-cursor-mask.png)";
+        element.current.style.maskSize = "400px";
+        element.current.style.maskRepeat = "no-repeat";
+        element.current.style.maskPosition = `${x - 200}px ${y - 200}px`;
+      }
+    }
+
+    document.addEventListener("mousemove", handler);
+    return () => document.removeEventListener("mousemove", handler);
+  }, []);
+  const className = `bg-[url(/images/bg-masked.png)] mouse-tracker fixed w-screen
+  h-screen z-10 top-0 left-0 opacity-90 bg-cover bg-no-repeat bg-fixed invisible`;
```

Comment:
use `cn` to break it down into multiple lines, much cleaner 🙏 

```suggestion
  const className = cn(
    "bg-[url(/images/bg-masked.png)] mouse-tracker fixed w-screen",
    "h-screen z-10 top-0 left-0 opacity-90 bg-cover bg-no-repeat bg-fixed invisible",
  );
```